src/three-components/SmoothControls.ts

```diff
@@ -503,7 +536,8 @@ export class SmoothControls extends EventDispatcher {
                 this[$lastTouches][0], this[$lastTouches][1]);
             const touchDistance =
                 this[$twoTouchDistance](touches[0], touches[1]);
-            const radiusDelta = -1 * (touchDistance - lastTouchDistance) / 10.0;
+            const radiusDelta = -1 * this[$zoomMeters] *
+                (touchDistance - lastTouchDistance) / 10.0;
```

Comment:
I don't know if this is actionable or not, but I think magic number `10.0` here might have been previously related to `FRAMED_HEIGHT`. Did you poke at that at all?

Reply:
Maybe, but it’s also an arbitrary scale. There’s an issue around dpr where we should probably fix this.